internal/pkg/acl/acl.go

```diff
@@ -198,13 +198,24 @@ func convertToFlags(operations ...any) string {
 			v = "cluster-scope"
 		}
 		s := strings.ToLower(strings.ReplaceAll(fmt.Sprint(v), "_", "-"))
-		ops = append(ops, fmt.Sprintf("`--%s`", s))
+		ops[i] = fmt.Sprintf("`--%s`", s)
 	}
 
 	sort.Strings(ops)
 	return strings.Join(ops, ", ")
 }
 
+func ConvertToLower(operations ...any) string {
+	ops := make([]string, len(operations))
+
+	for i, v := range operations {
+		ops[i] = strings.ReplaceAll(fmt.Sprint(v), "_", "-")
+	}
+
+	sort.Strings(ops)
```

Comment:
Why don't we sort the list before we pass it in? In both cases it's the same list every time.

Reply:
Yeah I can do that.

Reply:
I'm saying we should manually sort the strings on line internal/pkg/acl/acl.go:82 so we don't have to call this sort function every time the command is run.

Reply:
Ohhh, yeah I can do that.

Reply:
Done